tools/compile_tool/main.cpp

```diff
@@ -353,7 +353,7 @@ bool isFP32(const ov::element::Type& type) {
 static void setDefaultIO(ov::preprocess::PrePostProcessor& preprocessor,
                          const std::vector<ov::Output<ov::Node>>& inputs,
                          const std::vector<ov::Output<ov::Node>>& outputs) {
-    const bool isVPUX = FLAGS_d.find("VPUX") != std::string::npos;
+    const bool isVPUX = FLAGS_d.find("NPU") != std::string::npos;
```

Comment:
Please update device name from VPUX to NPU.

Reply:
Done.